plugins/inputs/docker/README.md

```diff
@@ -66,6 +68,26 @@ to gather stats from the [Engine API](https://docs.docker.com/engine/api/v1.24/)
 When using the `"ENV"` endpoint, the connection is configured using the
 [cli Docker environment variables](https://godoc.org/github.com/moby/moby/client#NewEnvClient).
 
+#### Permissions
+
+When using the default unix socket endpoint, telegraf must be given permission
+to access the socket. This can be done by adding telegraf to the docker group
```

Comment:
Maybe define "telegraf" as a user
> adding the telegraf user to the docker group

Reply:
I think all our packages already defines a `telegraf` user (and maybe also a `telegraf` group.), but I'll add a note that the user may need to be created if the packages are not used.

Reply:
It does already create that user. I was referencing the documentation and how it's assumed that a user knows that you are talking about the `telegraf` user, and not the `telegraf` binary in the docs. A bit pedantic, but there's a slight chance of confusion.

Reply:
Ah, yeah, I misunderstood. I'll call that out.